src/Tutor.Web/Controllers/Domain/DTOs/Feedback/EmotionsFeedbackDTO.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Tutor.Web.Controllers.Domain.DTOs.Feedback
+{
+    public class EmotionsFeedbackDto
+    {
+        public string Comment { get; set; }
```

Comment:
Still missing the learner ID. I imagine you will add it next? On the front end, this code should help https://github.com/Clean-CaDET/platform-tutor-ui-web/blob/master/src/app/modules/domain/learning-objects/question/question.service.ts#L24

Also to what do we want to tie this feedback? To a specific unit or KC?
Finally, we should include a DateTime to understand when the comment was made.